app/components/emoji_picker/index.js

```diff
@@ -0,0 +1,109 @@
+// Copyright (c) 2017-present Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import {connect} from 'react-redux';
+import {createSelector} from 'reselect';
+
+import {getCustomEmojisByName} from 'mattermost-redux/selectors/entities/emojis';
+
+import {getTheme} from 'app/selectors/preferences';
+import {CategoryNames, Emojis, EmojiIndicesByCategory} from 'app/utils/emojis';
+
+import EmojiPicker from './emoji_picker';
+
+const categoryToI18n = {
+    activity: {
+        id: 'mobile.emoji_picker.activity',
+        defaultMessage: 'ACTIVITY'
+    },
+    custom: {
+        id: 'mobile.emoji_picker.custom',
+        defaultMessage: 'CUSTOM'
+    },
+    flags: {
+        id: 'mobile.emoji_picker.flags',
+        defaultMessage: 'FLAGS'
+    },
+    foods: {
+        id: 'mobile.emoji_picker.foods',
+        defaultMessage: 'FOODS'
+    },
+    nature: {
+        id: 'mobile.emoji_picker.nature',
+        defaultMessage: 'NATURE'
+    },
+    objects: {
+        id: 'mobile.emoji_picker.objects',
+        defaultMessage: 'OBJECTS'
+    },
+    people: {
+        id: 'mobile.emoji_picker.people',
+        defaultMessage: 'PEOPLE'
+    },
+    places: {
+        id: 'mobile.emoji_picker.places',
+        defaultMessage: 'PLACES'
+    },
+    symbols: {
+        id: 'mobile.emoji_picker.symbols',
+        defaultMessage: 'SYMBOLS'
+    }
+};
+
+function fillEmoji(indice) {
+    const emoji = Emojis[indice];
+    return {
+        name: emoji.aliases[0],
+        aliases: emoji.aliases.join(',')
+    };
+}
+
+const getEmojiName = createSelector(
```

Comment:
Can you change this back now that the function does actually return full emojis and not just their names? Or maybe `getEmojisBySection` would be clearer since that's how it's formatted